test/core/ticktree/README.md

```diff
@@ -26,13 +26,13 @@ For each Mangrove operation that modifies the tick tree, there is a separate con
 
 Mangrove's tick tree is quite complex due to the different levels each being stored separately as well as some things being stored in `local`. There are there many scenarios to test, e.g, updating an offer to another tick within the same `Leaf` is different than updating it to a tick below a different `level1` position.
 
-## Tick scenarios
+## TickTreeIndex scenarios
```

Comment:
This file has some `TickTreeIndex` vestiges.
```suggestion
## Bin scenarios
```

Reply:
fixed in e3acc5dcc91a1869f8dc29f4aa80b61fa14dab2d